### ISSUE TITLE
Error: stops training after 5 minutes

### ISSUE THREAD

**Issue by Tylersuard** [chrisdonahue/wavegan#78]
I'm getting a warning from Tensorflow when I run this in Colab:

2020-03-26 21:52:54.799421: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:145] Filling up shuffle buffer (this may take a while): 3943 of 4096
2020-03-26 21:53:00.875418: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:195] Shuffle buffer filled.
WARNING:tensorflow:It seems that global step (tf.train.get_global_step) has not been increased. Current value (could be stable): 0 vs previous value: 0. You could increase the global step by passing tf.train.get_global_step() to Optimizer.apply_gradients or Optimizer.minimize.

This is after only 5 minutes of running the training.  For my dataset I put in around 5,000 audio files of 30 seconds in length.  What am I dong wrong?

Thank you for your help!

**Comment by evavdborn** [comment]
@Tylersuard Hi! I am getting the same error in Colab. How did you manage to fix it?

**Comment by JordanDavis-1** [comment]
Currently having the same issue. It works fine in google collab but in other environments having the same issue

**Comment by wxue1201** [comment]
Dear, how did it work out? Request help
